### PR TITLE
Added throttle Control based on vehicle classes and made it enabled by default.

### PR DESCRIPTION
**Pull request by monarchmatrix**
1) Made throttle Control to be enabled by default.
2) Added a check to apply Throttle control only on selected vehicle classes defined in Config file.
3) Changed the lshift key to disable the Control temporarily.

### CODE REVIEW

**Comment by Kuzkay** [comment]
Hey @monarchmatrix I appreciate the update, although I do not want to change the default functionality of the script. Could you put the default state (smoothing on/off) in the config?

**Comment by monarchmatrix** [comment]
@Kuzkay  
1)Added option to enable/disable smoothing by default in the config.
2) Config.keybinds will work accordingly depending on the Config.smoothingonkey configuration
3) If the vehicle class is disabled then it will be completely ignored from both the configs.

File: kq_throttlecontrol/client/client.lua [hunk 1]
```diff
@@ -35,11 +30,19 @@ local function TriggerThrottleControl()
 end
 
 RegisterCommand('+throttlecontrol', function()
-    enabled = false
+    if Config.smoothingonkey then
```
Comment (Kuzkay) [inline kq_throttlecontrol/client/client.lua]:
```lua
enabled = Config.smoothingOnKey
```

File: kq_throttlecontrol/client/client.lua [hunk 2]
```diff
@@ -35,11 +30,19 @@ local function TriggerThrottleControl()
...
+        enabled = true
+    else
+        enabled = false
+    end
 end, false)
 
 RegisterCommand('-throttlecontrol', function()
-    enabled = true
+    if Config.smoothingonkey then
```
Comment (Kuzkay) [inline kq_throttlecontrol/client/client.lua]:
```lua
enabled = not Config.smoothingOnKey
```

File: kq_throttlecontrol/client/client.lua [hunk 3]
```diff
@@ -1,6 +1,6 @@
 local maxRpm = Config.maxRpm
 local maxSpeed = Config.maxSpeed
-local enabled = true
+local enabled = not Config.smoothingonkey
```
Comment (Kuzkay) [inline kq_throttlecontrol/client/client.lua]: Codestyle
```lua
not Config.smoothingOnKey
```

File: kq_throttlecontrol/config.lua [hunk 4]
```diff
@@ -1,16 +1,38 @@
...
---- Max RPM in percentage
-Config.maxRpm = 0.2
+Config.maxRpm = 0.2                                -- Max RPM
 
---- Speed in KM/h
--- The throttle control will not affect the vehicles which drive over this speed
-Config.maxSpeed = 80
+Config.maxSpeed = 10                               -- The smoothing will not affect the vehicles which drive over this speed
+
+Config.smoothingonkey = true                       -- If true then Config.keybinds will enable smoothing and won't be applied by default, if false then will be applied by default and Config.keybinds will disable it.
```
Comment (Kuzkay) [inline kq_throttlecontrol/config.lua]: Codestyle
```lua
Config.smoothingOnKey
```

**Comment by monarchmatrix** [comment]
Improved the code styling for better understanding.

**Pull request merged**